### ISSUE TITLE
Lies!

### ISSUE THREAD

**Issue by camsaul** [metabase/metabase#1258]
```bash
$ docker run -d -p 3000:3000 —name metabase metabase/metabase:v0.12.0
2015/10/14 17:31:53 Post http:///var/run/docker.sock/v1.15/containers/create: dial unix /var/run/docker.sock: no such file or directory
```

**Comment by tnorthcutt** [comment]
FWIW, I just did a fresh install of docker (via homebrew), then what's suggested in the [README](https://github.com/metabase/metabase/blob/92ff9533e24a4a8247ffb570e172af3f60e6704c/README.md#docker) and got an error:
```
-> % docker run -d -p 3000:3000 —name metabase metabase/metabase
Unable to find image '—name:latest' locally
repository name component must match "[a-z0-9]+(?:[._-][a-z0-9]+)*"
```

**Comment by agilliland** [comment]
the issue there is a copy/paste problem i think.  the command should have dash dash name `--name` as the option not single-dash.

**Comment by tnorthcutt** [comment]
Yep, I just noticed that as well; pull request submitted: https://github.com/metabase/metabase/pull/1296

**Comment by agilliland** [comment]
fixed in #1361
